web_src/js/features/repo-legacy.js

```diff
@@ -89,10 +89,18 @@ export function initRepoCommentForm() {
     $('#comment-form').trigger('submit');
   });
 
-  const _promise = initComboMarkdownEditor($commentForm.find('.combo-markdown-editor'), {
-    onContentChanged(editor) {
-      $statusButton.text($statusButton.attr(editor.value().trim() ? 'data-status-and-comment' : 'data-status'));
-    },
+  // default markdown editor should have no more than one
+  $commentForm.find('.field.markdown .combo-markdown-editor').each((_, comboMarkdownEditor) => {
+    const _promise = initComboMarkdownEditor(comboMarkdownEditor, {
+      onContentChanged(editor) {
+        $statusButton.text($statusButton.attr(editor.value().trim() ? 'data-status-and-comment' : 'data-status'));
+      },
+    });
+  });
```

Comment:
This doesn't seem right.

`$statusButton` only works for issue comment, not for "new issue".

I will fix it later.

Reply:
Thanks for your help.